tests/python/unittest/test_pass_attrs_hash_equal.py

```diff
@@ -0,0 +1,29 @@
+import tvm
+
```

Comment:
Are there other sorts of fields that can appear in attributes classes that might also be good test cases? It might be good to have a representative sample for these tests.